mne/viz/misc.py

```diff
@@ -302,9 +302,22 @@ def plot_source_spectrogram(stcs, freq_bins, tmin=None, tmax=None,
 
 
 def _plot_mri_contours(mri_fname, surfaces, src, orientation='coronal',
-                       slices=None, show=True, show_indices=False,
-                       show_orientation=False, img_output=False, width=512):
-    """Plot BEM contours on anatomical slices."""
+                       slices=None, slices_as_figures=False, show=True,
+                       show_indices=False, show_orientation=False, width=512):
+    """Plot BEM contours on anatomical slices.
+
+    Parameters
+    ----------
+    slices_as_figures : bool
```

Comment:
I document the new param I added, but didn't feel the urge to add docs for all the other params there were already undocumented before I came along 😅